docs/TextArea.mdx

```diff
@@ -0,0 +1,94 @@
+---
+name: TextArea
+---
+
+import { Playground } from "docz"
+import { TextArea, Checkbox, Spacer, Serif, Flex } from "../src"
+import { Toggle } from "react-powerplug"
+
+## Basic
+
+<Playground>
+  <TextArea defaultValue="This is some text" onChange={console.log} />
+</Playground>
+
+## With error
+
+<Playground>
+  <Toggle initial={true}>
+    {({ on, toggle }) => (
+      <>
+        <TextArea
+          error={on ? "There was an error" : null}
+          defaultValue="This is some text"
+          onChange={console.log}
+        />
+        <Spacer mb={3} />
+        <Flex alignItems="center">
+          <Checkbox selected={on} onSelect={toggle} />
+          <Serif size="3">Show error</Serif>
+        </Flex>
+      </>
+    )}
+  </Toggle>
+</Playground>
+
+## With title
+
+<Playground title="With title">
+  <TextArea
+    title="Note"
+    error="There was an error"
+    defaultValue="This is some text"
+    onChange={console.log}
+  />
+</Playground>
+
+## With title and description
+
+<Playground>
+  <TextArea
+    title="Note"
+    description="For your security do not share personal information."
+    defaultValue="This is some text"
+    onChange={console.log}
+  />
+</Playground>
+
+## With character limit
+
+<Playground>
+  <TextArea
+    title="Note"
+    description="For your security do not share personal information."
+    defaultValue="This is some text"
+    characterLimit={200}
+    onChange={console.log}
+  />
+</Playground>
+
+## With character limit and error
+
+<Playground>
+
+    <Toggle initial={true}>
+    {({ on, toggle }) => (
+      <>
+        <TextArea
+          error={on ? "There was an error" : null}
+          defaultValue="This is some text"
+          characterLimit="322"
+          title="Further details"
+          description="Please provide any further pertinent details to help us assess your case."
+          onChange={console.log}
+        />
+        <Spacer mb={3} />
+        <Flex alignItems="center">
+          <Checkbox selected={on} onSelect={toggle} />
+          <Serif size="3">Show error</Serif>
+        </Flex>
+      </>
+    )}
+
+  </Toggle>
+  </Playground>
```

Comment:
@ds300 - I'll have more instructions once this is launched, but in the meantime can you move this file [to our new docs](https://github.com/artsy/palette/tree/master/www/content/docs/elements/inputs)? 

The only real difference is that the `<Playground>` component now takes a `title` prop: https://github.com/artsy/palette/blob/master/www/content/docs/elements/inputs/Checkbox.mdx

Reply:
I've got to leave the office now, but feel free to do this (anyone) if you want to get this merged tonight :+1:

Reply:
Its all good, can merge now.